### ISSUE TITLE
Cant install in nuxt 3.6.2.

### ISSUE THREAD

**Issue by captain-zsa** [uNmAnNeR/imaskjs#911]
When I install with `npm install vue-imask` – I see this error:
<img width="537" alt="Снимок экрана 2023-07-13 в 18 52 29" src="https://github.com/uNmAnNeR/imaskjs/assets/18168845/2254b4df-b4bd-4c97-9260-029430427ed0">

I use vite & nuxt:
```
nuxt-app@ project
├─┬ nuxt@3.6.2
│ ├─┬ @nuxt/vite-builder@3.6.2
│ │ ├─┬ @vitejs/plugin-vue-jsx@3.0.1
│ │ │ └── vue@3.3.4 deduped
│ │ ├─┬ @vitejs/plugin-vue@4.2.3
│ │ │ └── vue@3.3.4 deduped
│ │ └── vue@3.3.4 deduped
│ ├─┬ @unhead/vue@1.1.30
│ │ └── vue@3.3.4 deduped
│ ├─┬ unplugin-vue-router@0.6.4
│ │ └─┬ @vue-macros/common@1.4.0
│ │   └── vue@3.3.4 deduped
│ ├─┬ vue-router@4.2.4
│ │ └── vue@3.3.4 deduped
│ └─┬ vue@3.3.4
│   └─┬ @vue/server-renderer@3.3.4
│     └── vue@3.3.4 deduped
└─┬ pinia@2.1.4
  ├─┬ vue-demi@0.14.5
  │ └── vue@3.3.4 deduped
  └── vue@3.3.4 deduped
```

**Comment by captain-zsa** [comment]
I'm sorry, but error is repeated...

**Comment by uNmAnNeR** [comment]
@captain-zsa it's not released yet. Will be fixed in 7.1.3
